products/sourcerepo/terraform.yaml

```diff
@@ -25,7 +25,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "sourcerepo_repository_basic"
         primary_resource_id: "my-repo"
-        primary_resource_name: "fmt.Sprintf(\"my/repository%s\", context[\"random_suffix\"])"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-my/repository%s\", context[\"random_suffix\"])"
```

Comment:
this one i'm not sure we'll need either (will follow up to make sure) - if it had no dashes in the var name I didn't add the prefix but i might change that behavior

Reply:
The original test didn't fail, so reverting.

I got a little overzealous with my find & replace